test/smoke/src/main.ts

```diff
@@ -243,6 +246,8 @@ async function ensureStableCode(): Promise<void> {
 		return;
 	}
 
+	// {{SQL CARBON EDIT}}
+	/*
```

Comment:
Instead I'd just comment out the `ensureStableCode` call on line 293 (and add a reason why to the SQL CARBON EDIT when you do)